tesseract_motion_planners/src/trajopt_ifopt/problem_generators/default_problem_generator.cpp

```diff
@@ -55,6 +55,10 @@ DefaultTrajOptIfoptProblemGenerator(const std::string& name,
   const std::string& manipulator = composite_mi.manipulator;
   auto kin = request.env->getManipulatorManager()->getFwdKinematicSolver(manipulator);
   auto inv_kin = request.env->getManipulatorManager()->getInvKinematicSolver(manipulator);
+
+  // Synchronize the inverse kinematics with the forward kinematics
+  inv_kin->synchronize(fwd_kin);
```

Comment:
`error: ‘fwd_kin’ was not declared in this scope`
```suggestion
  inv_kin->synchronize(kin);
```
or I think we can keep it `fwd_kin` and change other instances to `fwd_kin` instead of `kin`